.github/workflows/causal_lm_cpp.yml

```
@@ -12,6 +12,87 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  cpp-beam_search_causal_lm-red-pajama-3b-instruct:
+    runs-on: ubuntu-20.04-16-cores
```

```suggestion
    runs-on: ubuntu-20.04
```
Can you try the smaller runner?